app/views/LivechatEditView.tsx

```diff
@@ -39,36 +43,87 @@ const styles = StyleSheet.create({
 	}
 });
 
-const Title = ({ title, theme }) =>
+interface ITitle {
+	title: string;
+	theme: string;
+}
+
+interface IField {
+	_id: string;
+	visibility: string;
+	scope: string;
+}
+
+interface IInputs {
+	[key: string]: string | string[] | undefined;
+	name: string;
+	email: string;
+	phone?: string;
+	topic: string;
+	tag: string[];
+}
+
+type TParams = IVisitor & IInputs;
+
+interface ILivechat extends ISubscription {
+	// Param dynamic depends on server
+	sms?: string;
+}
+
+interface IInputsRefs {
+	[index: string]: RNTextInput | null;
+	name: RNTextInput | null;
+	phone: RNTextInput | null;
+	topic: RNTextInput | null;
+}
+
+interface ICustomFields {
+	visitor?: { [key: string]: string };
+	livechat?: { [key: string]: string };
+}
+interface ILivechatEditViewProps {
+	// TODO: Refactor when migrate models
+	user: any;
+	navigation: StackNavigationProp<ChatsStackParamList, 'LivechatEditView'>;
+	route: RouteProp<ChatsStackParamList, 'LivechatEditView'>;
+	theme: string;
+	editOmnichannelContact: string[];
+	editLivechatRoomCustomfields: string[];
+}
+
+const Title = ({ title, theme }: ITitle) =>
 	title ? <Text style={[styles.title, { color: themes[theme].titleText }]}>{title}</Text> : null;
-Title.propTypes = {
-	title: PropTypes.string,
-	theme: PropTypes.string
-};
 
-const LivechatEditView = ({ user, navigation, route, theme, editOmnichannelContact, editLivechatRoomCustomfields }) => {
-	const [customFields, setCustomFields] = useState({});
-	const [availableUserTags, setAvailableUserTags] = useState([]);
+const LivechatEditView = ({
+	user,
+	navigation,
+	route,
+	theme,
+	editOmnichannelContact,
+	editLivechatRoomCustomfields
+}: ILivechatEditViewProps) => {
+	const [customFields, setCustomFields] = useState({} as ICustomFields);
```

Comment:
Why are you using it this way and not the default way?
Like that:
`const [customFields, setCustomFields] = useState<ICustomFields>({});`